tests/input_failures/missing_molecule_symbols.json

```diff
@@ -1,6 +1,6 @@
 {
-    "schema_name": "QC_JSON",
-    "schema_version": 0,
+    "schema_name": "qc_json",
```

Comment:
don't these have to be `qc_schema` or (`qcschema`, I'm confused)?

Reply:
These are input failures and apparently not well enough tested. Fixed.